src/utils/class-helpers.js

```diff
@@ -11,7 +11,7 @@ export function removeClass (el, className) {
 }
 
 export function hasClass (el, className) {
-  return el.className.split(/[\s]+/).includes(className)
+  return el.className.split(/[\s]+/).indexOf(className) > -1
```

Comment:
Also is there a reason we're using this for browsers that do support `classList`? @jpstevens